mesa/visualization/templates/js/PieChartModule.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+//Note: This pie chart is based off the example found here:
+//https://bl.ocks.org/mbostock/3887235
+
+var PieChartModule = function(fields, canvas_width, canvas_height) {
+    // Create the tag:
+    var svg_tag = "<svg width='" + canvas_width + "' height='" + canvas_height + "' ";
+    svg_tag += "style='border:1px dotted'></svg>";
+    // Append it to #elements
+    var svg_element = $(svg_tag)[0];
+    $("#elements").append(svg_element);
+
+    //create the legend
+    var legend_tag = "<div class='legend'></div>";
+    var legend_element = $(legend_tag)[0];
+    $("#elements").append(legend_element);
```

Comment:
I think it would be clearer to have a single "viz element" than the svg_element and the legend_element separately, but maybe there is a specific reason why you implemented it that way?